gamification-impl/src/main/resources/api-spec.yaml

```diff
@@ -226,6 +273,12 @@ components:
         WWW_Authenticate:
           schema:
             type: string
+  requestBodies:
+    patchBody:
+      content:
+        application/json:
+          schema:
+            $ref: ' '#/components/schemas/PatchBody'
```

Comment:
![image](https://user-images.githubusercontent.com/37512627/101377469-2588f980-38b2-11eb-85de-e7202c6c2448.png)
Error in syntax